### ISSUE TITLE
Documenter.jl Documentation?

### ISSUE THREAD

**Issue by haberdashPI** [JuliaAudio/SampledSignals.jl#37]
Just noticed that currently the documentation consists of the README and docstrings for functions.  Thought it might be useful to have an issue to track adding some docs with `Documenter.jl`. I imagine some of the work I've done for `Sounds.jl` can be pulled over and combined with some of the content in the README. 

Probably want to wait until the API is settled to work on this (and figured out which parts go in AudioIO.jl and which go in SampledSignals.jl).

**Comment by ssfrr** [comment]
Yeah, definitely 👍 to Documenter docs. Maybe once the API shakes out we can do a doc blitz.

It might be nice to combine the docs somehow to show how the different packages are used together (though still keeping API references for each package from the docstrings).  A while ago I registered http://juliaaudio.org and currently it just redirects to the GitHub organization, but that would be a nice place for the overview to live.
